temba/channels/types/telegram/type.py

```diff
@@ -46,11 +47,30 @@ def deactivate(self, channel):
         bot = telegram.Bot(config['auth_token'])
         bot.delete_webhook()
 
+    @classmethod
+    def get_quick_replies(cls, metadata, post_body):
+        metadata = json.loads(metadata)
+        quick_replies = metadata.get('quick_replies', None)
+        replies = []
+
+        if quick_replies:
+            for reply in quick_replies:
+                replies.append([dict(text=reply.get('title'))])
+
+            keyboard_json = dict(resize_keyboard=True, one_time_keyboard=True)
+            keyboard_json['keyboard'] = replies
+            post_body['reply_markup'] = json.dumps(keyboard_json)
```

Comment:
It's being done on backend side because is allowed to use flow expressions and doing that could cut or not complete the flow expression tag. Would you have any idea how do we can implement that limiting the characters but keep the flow expressions complete?

Reply:
Oh good point. I guess we just have to let those be truncated by the channels